docs/src/components/Overview/OverviewTile/styles.module.css

```diff
@@ -0,0 +1,44 @@
+.overviewTile {
+  display: flex;
+  flex-direction: row;
+  gap: 1.5rem;
+}
+
+.tileReversed {
+  flex-direction: row-reverse;
+}
+
+.contentContainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 1.5rem;
+  flex: 1;
+}
+
+.contentContainer h3 {
+  font-size: 32px;
+  line-height: 36.8px;
```

Comment:
Too deviated value for line-height 😄 Let's leave it as `36px`.

Reply:
It was taken from the designs, it is common to use it among our documentations, but i'll change it if'd like

Reply:
I think it shouldn't make a difference, I'm also of the opinion that final version does not have to be the same as designs 😄 `36px` would look better in code, at least imo